sdk/src/main/java/velocitas/sdk/middleware/Middleware.kt

```diff
@@ -0,0 +1,91 @@
+package velocitas.sdk.middleware
+
+import velocitas.getEnvVar
+
+abstract class Middleware protected constructor(
+    /**
+     * The type identifier of the concrete middleware implementation
+     */
+    val typeId: String
+) {
+    private val metadata: Metadata = Metadata()
+
+    /**
+     * Triggers the start of the middleware
+     */
+    open fun start() {
```

Comment:
```suggestion
    open fun start() { }
```

Or should this method be abstract?

Reply:
If I understood this correctly, then the CPP SDK provides empty methods as default, correct me if I'm wrong: 
https://github.com/eclipse-velocitas/vehicle-app-cpp-sdk/blob/main/sdk/include/sdk/middleware/Middleware.h#L59